src/internal/operations/backup.go

```diff
@@ -676,8 +677,15 @@ func mergeDetails(
 	mans []*kopia.ManifestEntry,
 	dataFromBackup kopia.DetailsMergeInfoer,
 	deets *details.Builder,
+	writeStats *kopia.BackupStats,
 	errs *fault.Bus,
 ) error {
+	// getting the values in writeStats before anything else so that we don't get a return from
+	// conditions like no backup data.
+	noMetaDataFile := deets.Details().DetailsModel.FilterMetaFiles().Entries
+	writeStats.TotalFileCount = len(noMetaDataFile)
```

Comment:
We probably actually want to use the `Items` call instead as that also filters out folders